app/controllers/admin/dashboard_controller.rb

```diff
@@ -26,7 +26,9 @@ def general
   end
 
   def problems
-    render_json_dump(problems: AdminDashboardData.fetch_problems(check_force_https: request.ssl?))
+    ProblemCheck.realtime.run
+
+    render json: { problems: serialize_data(AdminNotice.problem.all, AdminNoticeSerializer) }
```

Comment:
I think we can cache this endpoint for at least one minute to avoid running the realtime checks on every load of the admin page. WDYT? We can also have a flag to clear the cache that can be used by the "Recheck" button.